Client/Extensions/UIAlertControllerExtensions.swift

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+typealias UIAlertActionCallback = (UIAlertAction) -> Void
+
+// MARK: - Extension methods for building specific UIAlertController instances used across the app
+extension UIAlertController {
+
+    /**
+    Builds the Alert view that asks the user if they wish to opt into crash reporting.
+
+    - parameter sendReportCallback: Send report option handler
+    - parameter alwaysSendCallback: Always send option handler
+    - parameter dontSendCallback:   Dont send option handler
+    - parameter neverSendCallback:  Never send option handler
+
+    - returns: UIAlertController for opting into crash reporting after a crash occurred
+    */
+    class func crashOptInAlert(
+        sendReportCallback sendReportCallback: UIAlertActionCallback,
```

Comment:
I wanted to keep the first param to be explicit when calling the function so it's clear to the external caller that we are assigning the callback to the sendReport action. I know in Swift < 2.0 I could use the # sign to force the explicit first param but as far as I know in Swift 2.0 you just have to duplicate it :(